WebRoot/templates/mail/Message.template

```diff
@@ -713,3 +734,35 @@
 		</tr>
 	</table>
 </template>
+
+<!-- Collapsed mail addresses template -->
+<template id='mail.Message#CollapsedAddress'>
+	<table role="presentation">
+		<tbody>
+			<tr>
+				<td>
+					<$ var from = []; $>
+					<$ for(var i = 0; i < data.fromAddrs.length; i++) {
+						from.push(data.fromAddrs[i].getText());
+					} $>
+					<$= AjxStringUtil.fitString(from.join(", "), 200); $>
+				</td>
+				<td class="InfoLabel">
+					&nbsp;&nbsp;<$= ZmMsg.to $>&nbsp;&nbsp;
+				</td>
+				<td>
+					<$ var to = []; $>
+					<$ for(var i = 0; i < data.toAddrs.length; i++) {
```

Comment:
Well yes, but I just used this to be consistent with other code of templates. Let me know if you feels to need to change it.